beacon-chain/casper/validator.go

```diff
@@ -101,8 +101,8 @@ func GetAttestersTotalDeposit(attestations []*pb.AttestationRecord) uint64 {
 	return uint64(numOfBits) * params.DefaultBalance
 }
 
-// GetIndicesForSlot returns the attester set of a given slot.
-func GetIndicesForSlot(shardCommittees []*pb.ShardAndCommitteeArray, lcs uint64, slot uint64) (*pb.ShardAndCommitteeArray, error) {
+// getShardAndCommitteesForSlot returns the attester set of a given slot.
+func getShardAndCommitteesForSlot(shardCommittees []*pb.ShardAndCommitteeArray, lcs uint64, slot uint64) (*pb.ShardAndCommitteeArray, error) {
```

Comment:
I think this needs to be capitalized so other packages can use it. That's probably why the test failed.